scripts/minikube.sh

```diff
@@ -72,10 +72,33 @@ function install_minikube() {
     curl -Lo minikube https://storage.googleapis.com/minikube/releases/"${latest_version}"/minikube-linux-"${MINIKUBE_ARCH}" && chmod +x minikube && mv minikube /usr/local/bin/
 }
 
+function detect_kubectl() {
+    if type kubectl >/dev/null 2>&1; then
+        command -v kubectl
+        return
+    fi
+    # default if kubectl is not available
+    echo '/usr/local/bin/kubectl'
+}
+
 function install_kubectl() {
+    KUBE_LATEST_VERSION=$(curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt 2> /dev/null)
+    if [[ -z "${KUBE_LATEST_VERSION}" ]]; then
+        # skip: getting latest version failed, continue with what we have
+        KUBE_LATEST_VERSION=${KUBE_VERSION}
+    fi
```

Comment:
@nixpanic 
I think it started making sense to me, let me get some more clarification before I start tweaking this again.

1. Change s/KUBE_VERSION=${KUBE_VERSION:-"v1.14.10"}/KUBE_VERSION=${KUBE_VERSION:-"v1.18.3"}/g
2. May be also move the above block out of function install_kubectl() so that KUBE_LATEST_VERSION is available all the time.

Thoughts?

Reply:
In the top of the script, use `KUBE_VERSION=${KUBE_VERSION:-"latest"}`, similar to `MINIKUBE_VERSION`. Then, when installing kubectl, fetch the `latest` version that is in `stable.txt`.

No need to have `KUBE_LATEST_VERSION`, and the installation of a selected minikube version would be the same as selecting a version for kubectl.